### PR TITLE
toggleDeleteStudentConfirmation duplicates #5752

### LINKED ISSUE #5752: toggleDeleteStudentConfirmation duplicates
The same JS function toggleDeleteStudentConfirmation() is located in three places:
1. instructorSearch.js
2. instructorStudentList.js
3. instructorCourseDetails.js

I think it may be moved to the only instructor.js

### PR DESCRIPTION
**Pull request by belyabl9**
Fixes #5752

### CODE REVIEW

File: src/main/webapp/js/instructor.js [hunk 1]
```diff
@@ -50,6 +50,16 @@ function toggleDeleteFeedbackSessionConfirmation(courseID, name) {
 }
 
 /**
+ * Function that shows confirmation dialog for removing a student from a course
+ * @param studentName
+ * @param courseId
+ * @returns
```
Comment (YongJieYongJie) [inline src/main/webapp/js/instructor.js]: I think can remove the `@param`s and `@returns`

**Comment by YongJieYongJie** [comment]
@belyabl9 I was working on replacing browser dialog with Bootbox.js modals and was just about to change this :+1:

**Comment by belyabl9** [comment]
@YongJieYongJie , should I proceed with this PR adjusting it to Bootbox.js ?

**Comment by YongJieYongJie** [comment]
@belyabl9 I think it's better to stick to the issue because changing over to Bootbox.js modals cannot be considered as an side refactoring related to the issue

**Comment by belyabl9** [comment]
Ready to review

**Pull request merged**